app/controllers/activitypub/outboxes_controller.rb

```diff
@@ -6,12 +6,12 @@ class ActivityPub::OutboxesController < Api::BaseController
   include SignatureVerification
   include AccountOwnedConcern
 
+  before_action :require_signature!, if: :authorized_fetch_mode?
   before_action :set_statuses
   before_action :set_cache_headers
 
   def show
-    expires_in 1.minute, public: true unless page_requested?
-
+    expires_in page_requested? ? 0 : 3.minutes, public: !authorized_fetch_mode?
```

Comment:
while personally I appreciate ruby's lack of punctuation, others might find `expires_in(page_requested? 0 : 3.minutes, public: !authorized_fetch_mode?)` easier to read

Reply:
Done